inference_rules.adoc

```diff
@@ -127,6 +127,23 @@ MLCommons shall retain a library of past audit reports and send copies to MLComm
 
 An audit is expected to be completed within a 90 day period. Audits failing to meet this timeline can be requested to be invalidated by the auditee. The final decision to accept such a request will be taken by the Working Group.
 
+=== Assigned Peer Review Process
+
+In addition to general MLCommons review rules outlined https://github.com/mlperf/policies/blob/master/submission_rules.adoc[here], Inference workgroup uses Assigned Peer Review to improve scrutiny of the results. The goal is to ensure that each submission is reviewed. The process, executed by Results Chair, is as follows:
+
+* Each submitter will be assigned another submitter to review.
+
+* Chair will open a github issue against each company that has review assignment. Issues will be closed once reviewers indicate that they had finished their tasks. 
+
+* During the review, reviewers are asked to pay special attention to: (1) results validity, (2) methodology, (3) instructions for reproducibility, and (4) content of json files in systems directory.  
+
+* Any issues discovered are to be filed as github issues and resolved as usual. The issues should be filed before the deadline to raise objections.
+
+* Submitters are encouraged to review other submissions beyond their assigned review.
+
+During the Peer Review Process, a reviewer can ask for access to a submission system to reproduce the submission. If the request is granted and the reviewer is satisfied with the results, this can be used to waive the reviewed system from the audit, subject to the approval of the review group.
```

Comment:
I'm not sure this language is good, "If the request is granted and the reviewer is satisfied with the results, this can be used to waive the reviewed system from the audit"

Because a supplier could use an OEM to self review and then exempt themselves from the audit. I'd support language where the random audit is random.

Reply:
I like the idea of saving effort, but it seems too open ended and not specific. I'd favor simpler and easier rules.

Reply:
What if its waived from the selection audit but not the random audit for this round?

Reply:
@bitfort "subject to the approval of the review group" condition is still there. If the random audit is random, most of the time it is a waste of everyone's time like happening now. 

@peladodigital The waiver is meant for random audit. Because the selection audit is done by the review committee and automatically the committee will be eliminating uninteresting and already reproduced submissions.